frontend/src/components/VSourcesTable.vue

```diff
@@ -1,61 +1,85 @@
-          @keypress.enter="sortTable('media_count')"
-        >
-          <span class="flex w-full flex-row items-center justify-between">
-            {{ $t("sources.providers.item") }}
-            <TableSortIcon :active="sorting.field === 'media_count'" />
-          </span>
-        </th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr v-for="provider in sortedProviders" :key="provider.display_name">
-        <td>
-          <VLink :href="providerViewUrl(provider)">{{
-            provider.display_name
-          }}</VLink>
-        </td>
-        <td class="truncate font-semibold">
-          <VLink :href="provider.source_url">
-            {{ cleanSourceUrlForPresentation(provider.source_url) }}
-          </VLink>
-        </td>
-        <td class="text-right">
+  <div>
+    <table
+      :aria-label="$t('sources.aria.table').toString()"
+      role="region"
+      class="not-prose source-table w-full table-fixed text-base"
+    >
+      <thead>
+        <tr>
+          <th
+            tabindex="0"
+            @click="sortTable('display_name')"
+            @keypress.enter="sortTable('display_name')"
+          >
+            <span class="flex w-full flex-row items-center justify-between">
+              {{ $t("sources.providers.source") }}
+              <TableSortIcon :active="sorting.field === 'display_name'" />
+            </span>
+          </th>
+          <th
+            tabindex="0"
+            @click="sortTable('source_url')"
+            @keypress.enter="sortTable('source_url')"
+          >
+            <span class="flex w-full flex-row items-center justify-between">
+              {{ $t("sources.providers.domain") }}
+              <TableSortIcon :active="sorting.field === 'source_url'" />
+            </span>
+          </th>
+          <th
+            tabindex="0"
+            @click="sortTable('media_count')"
+            @keypress.enter="sortTable('media_count')"
+          >
+            <span class="flex w-full flex-row items-center justify-between">
+              {{ $t("sources.providers.item") }}
+              <TableSortIcon :active="sorting.field === 'media_count'" />
+            </span>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="provider in sortedProviders" :key="provider.display_name">
+          <td>
+            <VLink :href="providerViewUrl(provider)">{{
+              provider.display_name
+            }}</VLink>
+          </td>
+          <td class="truncate font-semibold">
+            <VLink :href="provider.source_url">
+              {{ cleanSourceUrlForPresentation(provider.source_url) }}
+            </VLink>
+          </td>
+          <td class="text-right">
+            {{ getLocaleFormattedNumber(provider.media_count || 0) }}
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <section role="region" class="mobile-source-table">
+      <article v-for="provider in sortedProviders" :key="provider.display_name">
```

Comment:
I don't totally understand the usage of `article`. What is the intention meant to communicate with it? In JAWS it adds article start/end announcements when navigating through the page, without any useful context to know what the article would be about.

It would help to add `title` to this element. JAWS then announces the title instead of just "article" which makes it possible to skim the sources list by moving through each article instead of needing to go two elements into each article just to find out the name of the sources.

```suggestion
      <article v-for="provider in sortedProviders" :key="provider.display_name" :title="provider.display_name">
```

This seemed to test well locally when I tried it with JAWS.

Reply:
Thank you for showing me how you do the accessibility tests. I used the article element as in my head each row, each site was it's own box, not a full section, just an article. It seemed better than using div but it you think it might be better for accessibility then I'm open to chaning it